src/physics/manufactured_solution.cpp

```diff
@@ -35,6 +35,14 @@ bool isfinite(Sacado::Rad::ADvar<Sacado::Fad::DFad<double>> value)
     return std::isfinite(static_cast<double>(value.val().val()));
 }
 
+template <int dim, typename real>
+inline real ManufacturedSolutionZero<dim,real>
```

Comment:
No, since the boundary conditions in physics are still based on the manufactured solution function. The manufacturedSolutionZero lets me put homogeneous Dirichlet boundary conditions.